### PR TITLE
Build schema data viz page skeleton

### PR DESCRIPTION
**Pull request by cmperez**

### CODE REVIEW

File: client/js/store/app_store.js [hunk 1]
```diff
@@ -68,7 +68,6 @@ var AppStore = assign({}, EventEmitter.prototype, {
   removeBucketPrefix: function(bucketArray) {
     var buckets = [];
     bucketArray.forEach(function(schema) {
-      schema = schema.substr(schema.indexOf("-") + 1);
```
Comment (dduugg) [inline client/js/store/app_store.js]: nice work. sorry for not catching this the first time around.

File: client/js/components/app.jsx [hunk 2]
```diff
@@ -25,9 +25,7 @@ var App = React.createClass({
     return (
       <div>
         <Navbar />
-        <div className={"wrapper"}>
-          <InterfaceComponent state={this.state} router={router}/>
-        </div>
+        <InterfaceComponent state={this.state} router={router}/>
```
Comment (zindlerb) [inline client/js/components/app.jsx]: :clap:

File: client/js/store/app_store.js [hunk 3]
```diff
@@ -68,7 +68,6 @@ var AppStore = assign({}, EventEmitter.prototype, {
   removeBucketPrefix: function(bucketArray) {
     var buckets = [];
     bucketArray.forEach(function(schema) {
```
Comment (dduugg) [inline client/js/store/app_store.js]: this would be more elegant with _.transform

**Comment by dduugg** [comment]
:+1:

**Comment by zindlerb** [comment]
:+1:

**Pull request merged**